examples/loop-arbitrary-sequential-steps.yaml

```
@@ -0,0 +1,49 @@
+# This example demonstrates a workflow accepting a list of items (as JSON string)
+# as an input parameter, and using it for expanding a step into a sequential task
+# of arbitrary steps. When one step fails, the iteration will break, stopping
+# following steps from running.
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: loop-arbitrary-sequential-steps-
+spec:
+  entrypoint: loop-arbitrary-sequential-steps-example
+  arguments:
+    parameters:
+    - name: step_params
+      value: |
+        [
+          { "exit_code": 0, "message": "succ 1" },
```

Minor: I think this is clearer
```suggestion
          { "exit_code": 0, "message": "succeeds 1" },
```

---

Sorry the forced push ruined the requested change. Is there anything else to fix?